pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -855,6 +858,249 @@ struct ContractionOpConversion
   }
 };
 
+struct ArgSortOpConversion : public OpConversionPattern<tile::ArgSortOp> {
+  using OpConversionPattern<tile::ArgSortOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(tile::ArgSortOp op, ArrayRef<Value> operands,
+                  ConversionPatternRewriter &rewriter) const override {
+    IVLOG(3, "ArgSortOpConversion::matchAndRewrite");
+    tile::ArgSortOpAdaptor adaptor(operands);
+    auto loc = op.getLoc();
+    auto i32Type = rewriter.getI32Type();
+    auto indexType = rewriter.getIndexType();
+
+    // Axis represents one of the tensor dimensions.
+    int64_t axisAttr = op.axis().getSExtValue();
+
+    // Special case value -1 indicates the last axis.
+    Value tensor = adaptor.tensor();
+    auto shape = tensor.getType().cast<MemRefType>().getShape();
+    size_t tensorDims = shape.size();
+    size_t axis = static_cast<size_t>(axisAttr);
+    if (-1 == axisAttr) {
+      axis = axisAttr + static_cast<int64_t>(tensorDims);
+    } else if (axisAttr < 0 || axis >= tensorDims) {
+      return failure();
+    }
```

Comment:
Not required, but it might be nice to do a full numpy-style negative axis support, where also -2 is the 2nd-to-last axis, -3 is 3rd-to-last, etc.